### PR TITLE
Add ocular statuses to members page

### LINKED ISSUE #65: Feature Request: ocular status
hello it's me business man jeffalo, i am here to suggest showing ocular status under bios on the user list.

**Is your feature request related to a problem?**
i like ocular status

**Describe alternatives you've considered**
n/a ocular is too original

**Additional context**
https://my-ocular.jeffalo.net/docs

### PR DESCRIPTION
**Pull request by FunctionalMetatable**
Resolves #65, maybe.

### CODE REVIEW

**Comment by Accio1** [comment]
Blocked because discussion on the original issue was not complete.

**Comment by FunctionalMetatable** [comment]
ok

File: forumhelpers/FHP.js [hunk 1]
```diff
@@ -1,3 +1,17 @@
+function getOcular(username) {
+	var ocular_pull = new XMLHttpRequest()
+	ocular_pull.open("GET", "https://my-ocular.jeffalo.net/api/user/" + username)
+	ocular_pull.send();
+	ocular_pull.onreadystatechange = function() {
+		if (ocular_pull.readyState == 4 && ocular_pull.status === 200) {
+			var ocular_data = JSON.parse(ocular_pull.responseText)
+			if (ocular_data.status) {
+				document.getElementById("ocular-status-" + username).innerHTML = ocular_data.status;
```
Comment (jeffalo) [inline forumhelpers/FHP.js:9]: do not use `innerHTML`. this opens the door for attackers to input html into an ocular status and commit xss.

Comment (jeffalo) [reply forumhelpers/FHP.js:9]: also its 2021, fetch is supported on most modern browsers now :)

**Review by jeffalo — CHANGES_REQUESTED** [review]

File: forumhelpers/FHP.js (hunk 1 above)
Comment (FunctionalMetatable) [reply forumhelpers/FHP.js]: thanks!

**Review by FunctionalMetatable — COMMENTED** [review]
i'm new to this github pull requests thing and i don't know what i just did

**Comment by FunctionalMetatable** [comment]
for some reason the github app pull just broke it, will be closing this pr

**Comment by FunctionalMetatable** [comment]
@Accio1 / @leahcimto can i make a new PR?

**Comment by leahcimto** [comment]
am i crazy or did i just get an email notification here from @FunctionalMetatable

**Comment by FunctionalMetatable** [comment]
> am i crazy or did i just get an email notification here from @FunctionalMetatable
> 
> @FunctionalMetatable, the answer to your question is that there was not enough discussion on #65 however there might be now so I would ask @Accio1 and wait to get the issue assigned to you.

i deleted the message just after it was sent lol

thanks!

**Pull request closed without merging**